<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{background-color: rgb(91, 91, 91);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>383</width>
      <height>347</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="Juego1">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego1{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego1:hover{
background-color:rgb(86, 71, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego1</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="Juego7">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego7{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego7:hover{
background-color:rgb(146, 131, 255);
 padding-top: 5px;
}



</string>
       </property>
       <property name="text">
        <string>Juego7</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="Juego2">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego2{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego2:hover{
background-color:rgb(96, 81, 255);
 padding-top: 5px;
}



</string>
       </property>
       <property name="text">
        <string>Juego2</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="Juego8">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego8{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego8:hover{
background-color:rgb(156, 141, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego8</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Juego5">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego5{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego5:hover{
background-color:rgb(126, 111, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego5</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Juego4">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego4{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego4:hover{
background-color:rgb(116, 101, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego4</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="Juego6">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego6{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego6:hover{
background-color:rgb(136,121, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego6</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Juego3">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Juego3{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#Juego3:hover{
background-color:rgb(106, 91, 255);
 padding-top: 5px;
}




</string>
       </property>
       <property name="text">
        <string>Juego3</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>801</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label{

font: 64pt &quot;Old London&quot;;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(216, 181, 0, 255), stop:0.447761 rgba(255, 255, 255, 255), stop:0.920398 rgba(222, 224, 50, 255));
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(178, 178, 178, 30), stop:0.462687 rgba(135, 135, 135, 50), stop:1 rgba(121, 121, 121, 30));
}
</string>
    </property>
    <property name="text">
     <string>Casino Virtual</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#imagen{background-color: rgb(100, 150, 100);}</string>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_recargar">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#_recargar{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#_recargar:hover{
background-color:rgb(146, 131, 255);
 padding-top: 5px;
}</string>
    </property>
    <property name="text">
     <string>Recargar</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_graphlayout"/>
   </widget>
   <widget class="QPushButton" name="_actualizar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#_actualizar{
selection-color: rgb(100, 255, 76);
font: 36pt &quot;Gloucester MT Extra Condensed&quot;;

color:rgb(235,235,235);
 background: rgb(155,155,155);
}
QPushButton#_actualizar:hover{
background-color:rgb(136,121, 255);
 padding-top: 5px;
}</string>
    </property>
    <property name="text">
     <string>Actualizar Gr√°fica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 255, 255);</string>
    </property>
    <property name="text">
     <string>Saldo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>490</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 255, 255);</string>
    </property>
    <property name="text">
     <string>Transacciones</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
